### PR TITLE
seminars: release

### PR DESCRIPTION
**Pull request by harunurhan**
INSPIR-3548

### CODE REVIEW

**Comment by harunurhan** [comment]
nope. we don't want the card yet apparently, probably will come later @StellaCh ?

File: ui/ui-tests/tests/submissions/literature.test.js [hunk 1]
```diff
@@ -36,7 +36,10 @@ describe('Literature Submission', () => {
     await selectDocType(page, 'article');
 
     const image = await takeScreenShotForDesktop(page);
-    expect(image).toMatchImageSnapshot();
+    expect(image).toMatchImageSnapshot({
```
Comment (monaawi) [inline ui/ui-tests/tests/submissions/literature.test.js:39]: why is this needed?

Comment (harunurhan) [reply ui/ui-tests/tests/submissions/literature.test.js:39]: it's flaky, has been feeling for a while here and there.
